File: ce/customer-engagement/on-premises/snippets/csharp/CRMV8/sharepointintegration/cs/retrieveabsoluteandsitecollectionurls2.cs
// <snippetretrieveabsoluteandsitecollectionurls2>


 // Retrieve the absolute URL and the Site Collection URL
 // of the SharePoint document location record.
 RetrieveAbsoluteAndSiteCollectionUrlRequest retrieveRequest = new RetrieveAbsoluteAndSiteCollectionUrlRequest
 {
     Target = new EntityReference(SharePointDocumentLocation.EntityLogicalName, _spDocLocId)
 };
 RetrieveAbsoluteAndSiteCollectionUrlResponse retriveResponse = (RetrieveAbsoluteAndSiteCollectionUrlResponse)_serviceProxy.Execute(retrieveRequest);

 Console.WriteLine("Absolute URL of document location record is '{0}'.", retriveResponse.AbsoluteUrl.ToString());
 Console.WriteLine("Site Collection URL of document location record is '{0}'.", retriveResponse.SiteCollectionUrl.ToString());

// </snippetretrieveabsoluteandsitecollectionurls2>

File: ce/customerengagement/on-premises/snippets/visualbasic/CRMV8/sharepointintegrationvb/vb/retrieveabsoluteandsitecollectionurls2.vb
' <snippetretrieveabsoluteandsitecollectionurls2>


 ' Retrieve the absolute URL and the Site Collection URL
 ' of the SharePoint document location record.
 Dim retrieveRequest As RetrieveAbsoluteAndSiteCollectionUrlRequest =
  New RetrieveAbsoluteAndSiteCollectionUrlRequest With {
   .Target = New EntityReference(SharePointDocumentLocation.EntityLogicalName, _spDocLocId)
  }
 Dim retriveResponse As RetrieveAbsoluteAndSiteCollectionUrlResponse =
  CType(_serviceProxy.Execute(retrieveRequest), RetrieveAbsoluteAndSiteCollectionUrlResponse)

 Console.WriteLine("Absolute URL of document location record is '{0}'.", retriveResponse.AbsoluteUrl.ToString())
 Console.WriteLine("Site Collection URL of document location record is '{0}'.", retriveResponse.SiteCollectionUrl.ToString())

' </snippetretrieveabsoluteandsitecollectionurls2>